### PR TITLE
Repackage client-blaze to blaze-client

### PR DESCRIPTION
**Pull request by krayoola**
Repackaging org.http4s.client.blaze -> org.http4s.blaze.client
Part of #2702

### CODE REVIEW

File: blaze-client/src/main/scala/org/http4s/blaze/client/BasicManager.scala [hunk 1]
```diff
@@ -15,11 +15,12 @@
  */
 
 package org.http4s
-package client
 package blaze
+package client
 
 import cats.effect._
 import cats.syntax.all._
+import org.http4s.client.{Connection, ConnectionBuilder, RequestKey}
```
Comment (rossabaker) [inline blaze-client/src/main/scala/org/http4s/blaze/client/BasicManager.scala:23]: `Connection` and `ConnectionBuilder` look like things that can be relocated, but that's for another ticket.

**Review by rossabaker — APPROVED** [review]
Nicely done!

**Comment by rossabaker** [comment]
Oh, this branch is broken pending #4830. I think the CI results are what we expect here.  Merging.

File: blaze-client/src/main/scala/org/http4s/blaze/client/BasicManager.scala (hunk 1 above)
Comment (krayoola) [reply blaze-client/src/main/scala/org/http4s/blaze/client/BasicManager.scala:23]: Sure, Seems like a chore do you have a ticket for that maybe i can also give it a try.

**Pull request merged**